atc/db/pipeline_test.go

```diff
@@ -1204,6 +1205,15 @@ var _ = Describe("Pipeline", func() {
 			Expect(err).ToNot(HaveOccurred())
 			Expect(found).To(BeFalse())
 		})
+
+		It("marks the pipeline ID in the deleted_pipelines table", func() {
+			pipeline.Destroy()
```

Comment:
Just to be extra safe

```suggestion
			err := pipeline.Destroy()
			Expect(err).ToNot(HaveOccurred())
```